app/instance.py

```diff
@@ -280,5 +280,14 @@ def ratelimit_handler(error):
     )
 
 
+@app.errorhandler(Exception)
```

Comment:
We are back to square one, now it'll catch every exception! The issue clearly says add a handler for custom exceptions defined by the app